README.md

```diff
@@ -76,10 +76,9 @@ More information about configuring the deployment credentials required for this
 ### `docker/login-action`
 
 The [`docker/login-action`](https://github.com/marketplace/actions/docker-login) action is used to authenticate calls
-to the user's Azure Container Registry, which will host the image that is then deployed to the Container App.
-Currently, `docker/login-action` is called during every invocation of this action, so the user's Azure Container
-Registry name is required, along with username and password credentials that are able to authenticate calls to this
-Azure Container Registry. These credentials are able to be retrieved by
+to the user's container registry, which will host the image that is then deployed to the Container App.
+Currently, `docker/login-action` is called during every invocation of this action, so the user's container registry base URL is required, along with username and password credentials that are able to authenticate calls to this
+container registry. These credentials are able to be retrieved by
 [creating a service principal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-service-principal#create-a-service-principal)

```

Comment:
Isn't this last part ACR only? I assume that for other Docker registries we're just going to pass credentials, right? Might be worth prefixing this part by something like "For Azure Container Registries ..." or something of that kind.